src/main/java/dev/hephaestus/glowcase/block/entity/EntityDisplayBlockEntity.java

```diff
@@ -0,0 +1,105 @@
+package dev.hephaestus.glowcase.block.entity;
+
+import dev.hephaestus.glowcase.Glowcase;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.Entity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.SpawnEggItem;
+import net.minecraft.nbt.NbtCompound;
+import net.minecraft.registry.RegistryWrapper;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Vec2f;
+import net.minecraft.world.World;
+
+public class EntityDisplayBlockEntity extends ItemDisplayBlockEntity {
```

Comment:
Why is this extending the item display block?

Reply:
I was wanting to reduce the amount of copy-pasting from the Item Display as both blocks(item and entity displays) acted mostly the same. I can change to not extend it and copy-paste everything needed from the item display for the entity display if wanted.